core/src/main/kotlin/gropius/service/architecture/RelationService.kt

```diff
@@ -75,6 +76,22 @@ class RelationService(
         return graphUpdater.save(relation, nodeRepository)
     }
 
+    /**
+     * Creates multiple [Relation]s based on the provided [input]
+     * Checks the authorization status
+     *
+     * @param authorizationContext used to check for the required permission
+     * @param input defines the [Relation]s
+     * @return the saved created [Relation]s
+     */
+    suspend fun bulkCreateRelation(
+        authorizationContext: GropiusAuthorizationContext, input: BulkCreateRelationInput
+    ): List<Relation> {
+        input.validate()
+        val relations = input.relations.map { createRelation(authorizationContext, it) }
+        return relations
```

Comment:
aren't most of the relations ded if you din't hand through the dfe?

Reply:
I don't really understand this comment. All mutations use the dfe just for authorization information

Reply:
brainfart. meant nodeCache and that's irrelevant, because nit saved